sentry-types/src/protocol/envelope.rs

```diff
@@ -256,13 +441,19 @@ impl From<Transaction<'static>> for Envelope {
 
 #[cfg(test)]
 mod test {
-    use std::time::{Duration, SystemTime};
+    use std::{
+        str::FromStr,
+        time::{Duration, SystemTime},
+    };
```

Comment:
This is just cosmetic really (and probably something that rustfmt/rust-analyzer should just autofix), throughout the rest of codebase, we prefer separate lines for `std` imports, instead of nesting them. Maybe the rust-analyzer default is set differently.
Doesn’t matter really, but we should at least be consistent.